.gitlab-ci.yml

```diff
@@ -88,9 +116,27 @@ Base:
           # - aws/rhel-8.4-aarch64
         INTERNAL_NETWORK: ["true"]
 
+Base-nightly:
+  stage: test
+  extends: .terraform
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule"'
+  variables:
+    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+  script:
+    - schutzbot/deploy.sh
+    - /usr/libexec/tests/osbuild-composer/base_tests.sh
```

Comment:
I don't like the fact that the scripts section is already different from what is executed in `Base`. IMO extend from Base and redefine only rules and runner sections, similar to how `New OSTree:` does it.